lib/fibonacci.rb

```diff
@@ -4,5 +4,6 @@
 # Space Complexity - ?  (should be O(n))
 # Hint, you may want a recursive helper method
 def fibonacci(n)
```

Comment:
Take another look at the notes on `fibonacci`, there's more than enough there to make a stab at this problem.